app/services/api/sectors/companies_count_grouped_by_scenario.rb

```diff
@@ -0,0 +1,46 @@
+module Api
+  module Sectors
+    class CompaniesCountGroupedByScenario
+      CP_SCENARIOS = %w[below_2 exact_2 paris not_aligned no_disclosure].freeze
+
+      # Returns info of how many Companies from each Sector are in given CP scenario group
+      #
+      # [
+      #  {
+      #   name: 'Below 2',
+      #   data: [ ['Coal Mining', 52], ['Steel', 73] ]
+      #  },
+      #  {
+      #    name: 'Paris',
+      #    data: [ ['Coal Mining', 65], ['Steel', 26] ]
+      #  }
+      # ]
+      def get
+        CP_SCENARIOS.map do |cp_alignment|
```

Comment:
this needs to be rewritten after https://github.com/Vizzuality/laws_and_pathways/pull/59 is merged